Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/CategoryUICreator.cs

```diff
@@ -169,28 +168,31 @@ private void SelectCategoryGroup(AssetType category)
 
         private void DefaultZoom()
         {
-            if (bodyType == BodyType.HalfBody)
-            {
-                cameraZoom.ToHalfBody();
-            }
-            else
+            switch (CoreSettingsHandler.CoreSettings.BodyType)
             {
-                cameraZoom.ToFullbodyView();
+                case BodyType.HalfBody:
+                    cameraZoom.ToHalfBody();
+                    break;
+                default:
+                    cameraZoom.ToFullbodyView();
+                    break;
             }
         }
 
         private void SwitchZoomByCategory(AssetType category)
         {
-            if (bodyType != BodyType.HalfBody)
+            if (CoreSettingsHandler.CoreSettings.BodyType == BodyType.HalfBody)
             {
-                if (category.IsOutfitAsset())
-                {
-                    cameraZoom.ToFullbodyView();
-                }
-                else
-                {
-                    cameraZoom.ToFaceView();
-                }
+                return;
+            }
+            
+            if (category.IsOutfitAsset())
+            {
+                cameraZoom.ToFullbodyView();
+            }
+            else
```

Comment:
the else is kinda redundant you an just add a return after 
cameraZoom.ToFullbodyView();